packages/widget/src/components/address-input/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './address-input';
```

Comment:
can we use named exports, those default exports have really weak support in IDEs